config.py

```diff
@@ -1,17 +1,26 @@
+import os
+import logging
 LOGGING_CONFIG = {
     'version': 1,
     'formatters': {'default': {
-        'format': '[%(asctime)s] %(levelname)s in %(module)s: %(message)s',
+        'format': '[%(asctime)s.%(msecs)03d] %(levelname)s in %(module)s: %(message)s',
+        'datefmt': '%d/%m/%Y %H:%M:%S'
     }},
     'handlers': {'console': {
         'class': 'logging.StreamHandler',
         'stream': 'ext://sys.stdout',
-        'formatter': 'default'
-    }, 'file': {
-        'class': 'logging.FileHandler',
         'formatter': 'default',
-        'filename': './log.log'
-    }
+        'level': 20
```

Comment:
what's level? Can give me a summary of what it does and a link for me to read about it?

Reply:
The levels give different importance to the message. Each handler has a different level that will filter messages not as important. So the console might not need to print as many messages as in the log file as otherwise it would clutter the space so we give it a higher level filter. 

The level I added is there as future functionality but no handler takes in that level yet.

https://docs.python.org/3/library/logging.html#logging-levels

Reply:
Since we want to set the level here, we should use constants defined by the logging library. Can just use `'INFO'` or `'DEBUG'` instead? 

See https://github.com/AI3SW/stargan-v2/blob/b81452e36732eb81b42431c98032038f16a665ec/config.py#L26